src/OrleansCodeGenerator/CodeGeneratorCommon.cs

```diff
@@ -310,5 +310,39 @@ public static string GetGeneratedNamespace(Type type, bool randomize = false)
 
             return result;
         }
+
+        public static TypeParameterConstraintClauseSyntax[] GetGenericTypeParameterConstraints(Type type)
```

Comment:
This seems to serve the same purpose as `SyntaxFactoryExtensions.GetTypeConstraintSyntax`